index.md

```diff
@@ -40,33 +40,6 @@ no_in_page_title: true
 <p class="lead text-center">
   Have questions, comments, or feedback? Want to set up a teleconference or in-person meeting?
   <br/>
-  <strong>Reach out to us at <a href="mailto:stix-taxii@hq.dhs.gov">stix-taxii@hq.dhs.gov</a>!</strong>
+  <strong>Reach out to us at <a href="mailto:cybox@mitre.org.">cybox@mitre.org</a>!</strong>
```

Comment:
Why was this changed (other than the awkwardness of contacting "stix-taxii" about CybOX)? I'm assuming this will change with the Oasis transition as well?